prowler/__main__.py

```diff
@@ -301,6 +302,9 @@ def prowler():
     if checks_folder:
         remove_custom_checks_module(checks_folder, provider)
 
+    # clean local directories
+    clean_provider_local_output_directories(args)
+
```

Comment:
Make this function compatible with all providers or do the condition `if args.output_bucket or args.output_bucket_no_assume` here